recipes-bsp/coreboot-utils/files/0001-Makefile-disable-deprecated-warnigs-as-errors.patch

```diff
@@ -0,0 +1,25 @@
+From 2c095fae49351a6be86934658e20de0e9f985061 Mon Sep 17 00:00:00 2001
+From: wkawka <wiktor.kawka@3mdeb.com>
+Date: Fri, 16 Sep 2022 11:09:29 +0200
+Subject: [PATCH] Makefile: disable deprecated warnings as errors
+
+Signed-off-by: wkawka <wiktor.kawka@3mdeb.com>
+---
+ Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index 7f38bd6b..e3615a11 100644
+--- a/Makefile
++++ b/Makefile
+@@ -124,7 +124,7 @@ endif
+ # Provide default CC and CFLAGS for firmware builds; if you have any -D flags,
+ # please add them after this point (e.g., -DVBOOT_DEBUG).
+ DEBUG_FLAGS := $(if ${DEBUG},-g -Og,-g -Os)
+-WERROR := -Werror
++WERROR := -Werror -Wno-deprecated-declarations
```

Comment:
```
cc1: note: unrecognized command-line option '-Wno-unknown-warning' may have been intended to silence earlier diagnostics
```

Reply:
@m-iwanicki what we can do to close this?

Reply:
This is not an error so we don't need to do anything about it. Or we could remove this patch if it's not needed.